pages/contribute/github_way.md

```diff
@@ -3,68 +3,74 @@ title: GitHub way
-
-1. If you change your mind about anything in your pull request and the request is not closed, or if the editor tells you to edit your request during the review process, you have to:
-    * Go to your pull request
-    * Click on "Files changed" in the top menu bar.
-      {% include image.html file="files_changed_github.png" inline=true alt="Files changed tab on GitHub" %}
-    * Click on the icon with 3 dots "..." of the file you  want to edit and then click on "Edit file".
-      {% include image.html file="3_dots_github.png" inline=true alt="File change options on GitHub" %}
-    * Make your changes.
-    * Click on “Commit changes”.
-
-{% include callout.html type="note" content="Anyone can comment on your issue or pull request and you can reply. For more information on this, please visit the [GitHub documentation](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)" %}
-
-## Request a review
-
-If you open a normal pull request then a review is automatically requested. The relevant editors will check your changes. If your request is still in draft, click on "Ready for review" to request a review. You can find more information about draft pull requests in the [GitHub documentation](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request#marking-a-pull-request-as-ready-for-review).
-
-## Address editors' comments
-
-1. When editors add comments or add a review of your pull request, you will be notified.
-1. You need to address editors' comments and requests by editing your pull request as in step 7 (see above).
-   * Go to your pull request
-   * Click on "Files changed" in the top menu bar.
-     {% include image.html file="files_changed_github.png" inline=true alt="Files changed tab on GitHub" %}
-   * Click on the icon with 3 dots "..." of the file you  want to edit and then click on "Edit file".
-     {% include image.html file="3_dots_github.png" inline=true alt="File change options on GitHub" %}
-   * Make your changes.
-   * Click on “Commit changes”.
-
-1. When all the requests have been addressed, the editors will mark the conversation as "Resolved" and the proposed changes as "Approved".
-1. You content is ready to be merged and published in the main website.
-1. Editors publish your content.
+2. Click “Issues” in the top menu bar and look at the existing issues. See if your idea or suggestion is already being discussed.
+    * If an issue exists, comment and let people know if you want to contribute.
+    * If no relevant issue exists, create a new issue by clicking the green “New issue” button on the right and choosing one of the issue templates. You can find more information on creating issues in the [GitHub documentation](https://docs.github.com/en/github/managing-your-work-on-github/creating-an-issue).
```

Comment:
```suggestion
    * If no relevant issue exists, create a new one: click the green “New issue” button on the right and choose a template. You can find more information on creating issues in the [GitHub documentation](https://docs.github.com/en/github/managing-your-work-on-github/creating-an-issue).
```

Reply:
I implemented @martin-nc's suggestion above, as it is clearer. I made additional modifications to avoid repeating the word "issue" too often.

Reply:
Woops didn't see this comment, I've also implemented it !